CMakePresets.json

```diff
@@ -714,6 +723,17 @@
       "configurePreset": "cudax-cpp20",
       "inherits": "cudax-base"
     },
+    {
+      "name": "cudax-cpp20-cufile",
+      "displayName": "Run only cuFile tests for cudax C++20",
+      "configurePreset": "cudax-cpp20",
+      "inherits": "cudax-base",
+      "filter": {
+        "include": {
+          "name": "^cudax\\.cpp20\\.test\\.cufile.*$"
+        }
+      }
+    },
```

Comment:
This isn't needed -- we don't define per-test presets. If only a specific preset is wanted, just run it with CTest. These test presets only exist for our CI scripts.

If you want to have presets in your local builds, you can define a CMakeUserPresets.json locally.

```suggestion
```